samples/ftest/TestTask.cpp

```diff
@@ -51,29 +52,37 @@ void TestTask::operator()() {
             if(_line == "trigger_redis_list") {
                 DARWIN_LOG_DEBUG("TestTask:: triggered redis list action");
                 if(REDISAddList(_redis_list, _line)) {
-                    _certitudes.push_back(0);
+                    _packet.AddCertitude(0);
                 }
                 else {
-                    _certitudes.push_back(DARWIN_ERROR_RETURN);
+                    _packet.AddCertitude(DARWIN_ERROR_RETURN);
                 }
             }
             else if(_line == "trigger_redis_channel") {
                 DARWIN_LOG_DEBUG("TestTask:: triggered redis channel action");
                 if(REDISPublishChannel(_redis_channel, _line)) {
-                    _certitudes.push_back(0);
+                    _packet.AddCertitude(0);
                 }
                 else {
-                    _certitudes.push_back(DARWIN_ERROR_RETURN);
+                    _packet.AddCertitude(DARWIN_ERROR_RETURN);
                 }
             }
-            else {
+            else if (_line == "load_test") {
+                DARWIN_LOG_DEBUG("TestTask:: load_test triggered, doing work for some times (2 000 000 mults)");
+                int n = _line.size();
+                for(int i=0; i< 2000000;i++){
+                    n *=i;
+                }
+                _packet.AddCertitude(n & 1);
```

Comment:
I think I wanted to simulate a high CPU load to avoid optimization by the CPU (to test worst case scenarios without a specific filter), I can switch to a sleep